packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -125,6 +129,15 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
         const virtualizer = this.virtualizer!;
         this.visibleItems = virtualizer.getVirtualItems();
         this.scrollHeight = virtualizer.getTotalSize();
+        this.isScrolling = virtualizer.isScrolling;
+        // There are multiple browser bugs that can result in us getting stuck thinking that we're scrolling.
```

Comment:
Now that you've got bug reports, could you link to them from here?

Alternatively we could also file a Nimble issue that links to the browser issues and mark it blocked, then link to the Nimble issue. That probably gives us a slightly better chance of removing this workaround someday.